tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -271,7 +271,7 @@ def build_jobspec(self):
                     cmake_arch_option = '-DOPENSSL_NO_ASM=ON'
                 return create_docker_jobspec(
                     self.name,
-                    'tools/dockerfile/grpc_artifact_linux_%s' % self.arch,
+                    'tools/dockerfile/grpc_artifact_centos6_%s' % self.arch,
```

Comment:
should we get rid of https://github.com/grpc/grpc/blob/master/tools/dockerfile/grpc_artifact_linux_x64/Dockerfile entirely and build all the artifacts (that are currently built with it) using manylinux2010 based image?

Reply:
Not really because it's also being used for packaging and [build_package_python](https://github.com/grpc/grpc/blob/master/tools/run_tests/artifacts/build_package_python.sh) depends on `apt-get` which is not available on CentOS. But this can be revised not to rely upon `apt-get`, though.

Reply:
One minor stuff related with .NET is that .net core supports CentOS from 7 or later. This doesn't affect entire gRPC .net build process because .so file will be built at the stage of build_artifact but might affect the packaging process.